.github/workflows/nrm-ci.yml

```diff
@@ -0,0 +1,47 @@
+name: nrm-CI
+on: [push, pull_request]
+jobs:
+    nrm-ci:
+
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                python-version: [3.8]
+                check: [style, license, install]
+
+        env:
+            TERM: xterm-256color
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        defaults:
+            run:
+                shell: bash -l {0}
+
+        steps:
+        - uses: actions/checkout@v2
+        - name: Setup conda - Python ${{ matrix.python-version }}
+          uses: conda-incubator/setup-miniconda@v2
+          with:
+            activate-environment: condaenv
+            miniconda-version: "latest"
+            python-version: ${{ matrix.python-version }}
+            channels: conda-forge
+            channel-priority: flexible
+            auto-update-conda: true
+
+        - name: Style check
+          if: matrix.check == 'style'
+          run: |
+            pip install flake8
+            flake8
```

Comment:
Any specific reason to use the pip one instead of the ubuntu package?

Reply:
I was thinking installing via pip was satisfactory since this is completely in user-space and flake8 is entirely Python. Are there advantages to the ubuntu package that I'm unaware of?

Reply:
the system package probably doesn't get updated as much, which avoids problems like a new version triggering new tests, but I guess we don't really expect flake rules to change that much so probably not an issue.